malariagen_data/pf7.py

```diff
@@ -0,0 +1,314 @@
+            "FILTER_VQSRTrancheINDEL99.95to100.00+": [DIM_VARIANT],
+            "FILTER_VQSRTrancheINDEL99.95to100.00": [DIM_VARIANT],
+            "FILTER_VQSRTrancheSNP99.50to99.60": [DIM_VARIANT],
+            "FILTER_VQSRTrancheSNP99.60to99.80": [DIM_VARIANT],
+            "FILTER_VQSRTrancheSNP99.80to99.90": [DIM_VARIANT],
+            "FILTER_VQSRTrancheSNP99.90to99.95": [DIM_VARIANT],
+            "FILTER_VQSRTrancheSNP99.95to100.00+": [DIM_VARIANT],
+            "FILTER_VQSRTrancheSNP99.95to100.00": [DIM_VARIANT],
+            "FS": [DIM_VARIANT],
+            "ID": [DIM_VARIANT],
+            "InbreedingCoeff": [DIM_VARIANT],
+            "LOF": [DIM_VARIANT],
+            "MLEAC": [DIM_VARIANT, DIM_ALT_ALLELE],
+            "MLEAF": [DIM_VARIANT, DIM_ALT_ALLELE],
+            "MQ": [DIM_VARIANT],
+            "MQRankSum": [DIM_VARIANT],
+            "NEGATIVE_TRAIN_SITE": [DIM_VARIANT],
+            "NMD": [DIM_VARIANT],
+            "POSITIVE_TRAIN_SITE": [DIM_VARIANT],
+            "QD": [DIM_VARIANT],
+            "QUAL": [DIM_VARIANT],
+            "RAW_MQandDP": [DIM_VARIANT, DIM_PLOIDY],
+            "ReadPosRankSum": [DIM_VARIANT],
+            "RegionType": [DIM_VARIANT],
+            "SOR": [DIM_VARIANT],
+            "VQSLOD": [DIM_VARIANT],
+            "altlen": [DIM_VARIANT, DIM_ALT_ALLELE],
+            "culprit": [DIM_VARIANT],
+            "set": [DIM_VARIANT],
+        }
+
+    def _load_config(self, data_config):
+        """Load the config for data structure on the cloud into json format."""
+        if not data_config:
+            working_dir = os.path.dirname(os.path.abspath(__file__))
+            data_config = os.path.join(working_dir, "pf7_config.json")
+        with open(data_config) as pf7_json_conf:
+            config_content = json.load(pf7_json_conf)
+        return config_content
+
+    def sample_metadata(self):
+        """Access sample metadata and return as pandas dataframe
+
+        Returns
+        -------
+            df (pandas.DataFrame) : A dataframe of sample metadata on the samples that were sequenced as part of this resource.
+                                    Includes the time and place of collection, quality metrics, and accesion numbers.
+                                    One row per sample.
+
+        Example
+        -------
+        Access metadata as pandas dataframe:
+            >>> pf7.sample_metadata()
+        """
+        if self._cache_sample_metadata is None:
+            path = os.path.join(self._path, self.CONF["metadata_path"])
+            with self._fs.open(path) as f:
+                self._cache_sample_metadata = pd.read_csv(f, sep="\t", na_values="")
+        return self._cache_sample_metadata
+
+    def open_variant_calls_zarr(self):
+        """Open variant calls zarr
+
+        Returns
+        -------
+            root (zarr.hierarchy.Group): Root of zarr containing information on variant calls.
```

Comment:
Please edit to [numpydoc style](https://numpydoc.readthedocs.io/en/latest/format.html#returns).

Reply:
Should now all be in numpydoc style